rotkehlchen/externalapis/etherscan.py

```diff
@@ -372,6 +374,19 @@ def get_transactions(
                                 transactions=[internal_tx],
                                 relevant_address=None,  # can't know the address here
                             )
+
+                        dbevents = DBHistoryEvents(self.db)  # type: ignore[arg-type]
+                        with self.db.user_write() as write_cursor:  # type: ignore[union-attr]
+                            # Redecode genesis tx events
+                            dbevents.delete_events_by_tx_hash(
+                                write_cursor=write_cursor,
+                                tx_hashes=[GENESIS_HASH],
+                                chain_id=self.chain.to_chain_id(),  # type: ignore[arg-type]
```

Comment:
nitpick: I'm always afraid of this type ignores because they assume conditions that can change in the future. Maybe you can add a check for the case where chain is in `EVM_CHAIN_IDS_WITH_TRANSACTIONS_TYPE`.

Checking it seems that etherscan is not called with `SUPPORTED_EVM_CHAINS` but with `EVM_CHAIN_IDS_WITH_TRANSACTIONS` but even after changing the type there you will have another issue because the type of `to_chain_id` is not compatible with `EVM_CHAIN_IDS_WITH_TRANSACTIONS_TYPE`. Just leave it as it is then